docs/tutorial/3/README.md

```diff
@@ -31,6 +31,14 @@ available. By default, it downloads the videos at very low quality, and it also
 retrieves a 'meta.json' file regarding the video's provenance and other meta
 information.
 
+
+NOTE: This step currently doesn't work, and, until we get a better Docker image up, you'll need to use a workaround. The installation as-is uses an old version of youtube-dl and cannot properly fetch videos. To fix this, you need to:
+
+1. Run ``./mtriage dev --yaml docs/tutorial/3/3a.yaml`` to open a shell in the terminal with this configuration.
+2. Run ``pip3 install yt-dlp``. It may give you an error message, but it's likely this is okay.
+3. Check that yt-dlp has installed correctly. Run ``pip3 show yt-dlp`` and you should see a version installed. (Also okay if this command fails).
+4. Run ``python3 src/run.py`` which should work properly.
+
```

Comment:
Instead of adding this to the README, I would prefer to simply update the mtriage dependencies, Docker image and youtube selector to use yt-dlp.

It looks below like you've basically done this. Is there anything stopping us from just updating the official image in Docker hub with this new build from your perspective? (I know that neither the process of building a new mtriage image, nor of uploading it to Docker Hub manually or through CI, is documented: so this is something we can do together.)